staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -188,6 +245,10 @@ func (p *restrictedProfile) Apply(pod *corev1.Pod, containerName string, target
 
 	switch style {
 	case podCopy:
+		p.keeps.removeLabels(pod)
```

Comment:
As far as I understand, this is a missing bit for restricted profile and netadmin profile and sysadmin profile. Because I don't know any reason of why we don't clear labels and probes in these profiles. And it seems that we are also fixing this.